packages/browser/src/tracing/resource-timing.ts

```diff
@@ -0,0 +1,39 @@
+import type { Span } from '@sentry/core';
+import { browserPerformanceTimeOrigin } from '@sentry/core';
+import { extractNetworkProtocol } from '@sentry-internal/browser-utils';
+
+function getAbsoluteTime(time: number = 0): number {
+  return ((browserPerformanceTimeOrigin() || performance.timeOrigin) + time) / 1000;
+}
+
+/**
+ * Converts a PerformanceResourceTiming entry to span data for the resource span.
+ *
+ * @param resourceTiming
+ * @returns An array where the first element is the attribute name and the second element is the attribute value.
+ */
+export function resourceTimingToSpanAttributes(
+  resourceTiming: PerformanceResourceTiming,
+): Array<Parameters<Span['setAttribute']>> {
+  const timingSpanData: Array<Parameters<Span['setAttribute']>> = [];
+  if (resourceTiming.nextHopProtocol != undefined) {
```

Comment:
```suggestion
  if (resourceTiming.nextHopProtocol) {
```

I assume false/''/0 are not expected values here, so we can safe a teeny bit of bundle size?

Reply:
nevermind, empty string is a possible value (see https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/nextHopProtocol#value).